parser-typechecker/src/Unison/Typechecker/TypeVar.hs

```diff
@@ -10,6 +10,7 @@ import           Unison.Type (Type)
 import           Unison.Var (Var)
 import qualified Unison.Var as Var
 
+-- note: polarity doesn't factor into equality or ordering
```

Comment:
Guessing this comment is no longer relevant?

Reply:
Yeah, if you notice any of these, point them out.